docs/quick-start.md

```diff
@@ -41,7 +41,7 @@ simple-pipeline-cat-0-kqrhy                  2/2     Running     0          10s
 simple-pipeline-in-0-rhpjm                   1/1     Running     0          11s
 
 # Watch the log for the `output` vertex
-kubectl logs -f simple-pipeline-out-0-xxxx main
+kubectl logs -f simple-pipeline-out-0-xxxx numa
```

Comment:
Removing the container name from the command line is better and works - there's a period of time before this change gets released (however the doc would have been published), and someone follows this guide will fail.